security/acls/acl.sh

```diff
@@ -123,11 +121,11 @@ if [[ $? != 0 ]]; then
   exit 1
 fi
 
-echo -e "\n# Verify cluster is a Confluent Cloud Enterprise cluster"
+echo -e "\n# Verify cluster supports ACLs"
```

Comment:
Do we need this check any more?

Reply:
I'm not sure.  I don't know if it's possible for a cluster to exist that will fail this check across all environments.

Reply:
Please double check with CCloud engineering team on this but IIUC, all ccloud clusters now support ACLs.

Reply:
https://docs.confluent.io/current/cloud/limits.html#cloud-limits

We should assume it's available on all clusters

Reply:
@ybyzek removed